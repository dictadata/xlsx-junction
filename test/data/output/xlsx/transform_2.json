[
{"fie":"where's fum?","dt_date":"2018-10-07T05:00:00.000Z","foo":"first","bar":"row","baz":123,"fum":"here"},
{"fie":"where's fum?","dt_date":"2018-10-07T05:00:00.000Z","foo":"second","bar":"row","baz":456,"fum":"here"}
]