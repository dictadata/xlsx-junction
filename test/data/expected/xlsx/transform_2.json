[
{"fie":"where's fum?","Dt Test":"dt_date","Foo":"foo","Bar":"bar","Baz":"baz","Fobe":"fobe"},
{"fie":"where's fum?","Dt Test":"dt_date","Foo":"foo","Bar":"bar","Baz":"baz","Fobe":"fobe"}
]